service/src/routes/store.ts

```diff
@@ -0,0 +1,55 @@
+import express, { Request, Response, NextFunction } from 'express';
+import { StatusCodes } from 'http-status-codes';
+import * as admin from 'firebase-admin';
+
+const router = express.Router();
+
+admin.initializeApp({
+  credential: admin.credential.applicationDefault(),
+  //databaseURL: 'https://dapp-store-7f9ab.firebaseapp.com'
+  databaseURL: 'https://dapp-store-7f9ab-default-rtdb.europe-west1.firebasedatabase.app/'
```

Comment:
We should store this as an environmental variable

Reply:
I agree. If I understood correctly to define env. variables we need firebase functions feature enabled which requires paid plan. This leads me to a question, are we going to have Astar firebase account?

Reply:
Probably yes. Let’s create one (or use an alternative) when we deploy the dApps Store to Shiden after the Shibuya test. Since for Shibuya, we’ll only need to save contracts from a whitelist.